timepicker-library/src/main/java/com/erz/timepicker_library/TimePicker.java

```diff
@@ -339,4 +343,11 @@ public void setTime(Date date){
     public void setTimeChangedListener(OnTimeChangedListener timeChangedListener){
         this.timeChangedListener = timeChangedListener;
     }
+
+    public void setStep(int step) {
```

Comment:
would need an invalidate call at the end of this function in order to redraw the clock